README.Rmd

```diff
@@ -53,6 +53,7 @@ c("red","blue") %>% brightness(0.6) %>% saturation(seq(0,1,0.25)) %>% swatch
 
 This operation takes the original two colours, reduces their brightness to 60%, assigns a whole series of saturation levels to the result, and then passes it to `swatch` for visualisation. Notice that the pipeline is combinative (like the base function `outer`), returning each combination of parameters in a multidimensional array. The final shades are arranged in two rows by `swatch`, for convenience.
 
+### dichromat function and colour-blindness
```

Comment:
I'd rather this was removed for now, as it doesn't apply to everything below it

Reply:
Fine by me, I'll take it out!